uaclient/jobs/__init__.py

```diff
@@ -0,0 +1,14 @@
+from uaclient import config, util
+from uaclient.clouds.identity import get_cloud_type
+
+
+def enable_license_check_if_applicable(cfg: config.UAConfig):
+    series = util.get_platform_info()["series"]
+    if "gce" in get_cloud_type() and util.is_lts(series):
+        cfg.write_cache("marker-license-check", "")
+
+
+def disable_license_check_if_applicable(cfg: config.UAConfig):
+    if cfg.cache_key_exists("marker-license-check"):
+        cfg.delete_cache_key("marker-license-check")
+        util.subp(["systemctl", "stop", "ua-license-check.timer"])
```

Comment:
@blackboxsw good point!
First, more context on how we got here:

on everything except gcp focal:
```bash
touch /var/lib/ubuntu-advantage/marker-license-check
# timer has started
rm /var/lib/ubuntu-advantage/marker-license-check
systemctl stop ua-license-check.timer
# timer has stopped
touch /var/lib/ubuntu-advantage/marker-license-check
# timer has started
```

on gcp focal (the problem platform):
```bash
touch /var/lib/ubuntu-advantage/marker-license-check
# timer has started
rm /var/lib/ubuntu-advantage/marker-license-check
systemctl stop ua-license-check.timer
# timer has stopped
touch /var/lib/ubuntu-advantage/marker-license-check
# timer has NOT started !!!!
```
To work on gcp focal you need to put the `stop` first. This works:
```bash
touch /var/lib/ubuntu-advantage/marker-license-check
# timer has started
systemctl stop ua-license-check.timer
rm /var/lib/ubuntu-advantage/marker-license-check
# timer has stopped
touch /var/lib/ubuntu-advantage/marker-license-check
# timer has started
```

But of course that doesn't work anywhere else. On other platforms the following happens when you run `stop` first:
```bash
touch /var/lib/ubuntu-advantage/marker-license-check
# timer has started
systemctl stop ua-license-check.timer
rm /var/lib/ubuntu-advantage/marker-license-check
# timer has NOT stopped !!! (because it got immediately restarted before I rm'd the marker file
```

So the following will work on all platforms including gcp focal:
```bash
touch /var/lib/ubuntu-advantage/marker-license-check
# timer has started
systemctl stop ua-license-check.timer
rm /var/lib/ubuntu-advantage/marker-license-check
systemctl stop ua-license-check.timer
# timer has stopped
touch /var/lib/ubuntu-advantage/marker-license-check
# timer has started
```
:tada: 


Now, the wild thing is that the change you suggested is not necessary (!?). I know this because the following snippet from `license_check.feature` works on all platforms including gcp focal:
```gherkin
        # verify attach disables it
        When I run `sleep 5` with sudo
        When I attach `contract_token` with sudo
        Then I verify the `ua-license-check` systemd timer is disabled
        # verify detach enables it
        When I run `ua detach --assume-yes` with sudo
```

I'm not totally sure why and I'm open to making the change you suggested just to be safe though. Thoughts?

(I'm also going to rerun all of the integration tests to re-verify all of this)